colossalai/auto_parallel/meta_profiler/constants.py

```diff
@@ -1,5 +1,16 @@
+import operator
```

Comment:
This file may be integrated with the constants.py in tensor_shard folder.

Reply:
Sure, that will be better